cluster/store/store_snapshot.go

```diff
@@ -1,30 +1,33 @@
 package store
 
 import (
+	"errors"
+	"github.com/ByteStorage/FlyDB/config"
 	"github.com/hashicorp/raft"
-	"io"
+	"os"
+	"path"
 )
 
-// snapshot implements raft.SnapshotStore interface
-type snapshot struct {
-	//implement me
+func newSnapshotStore() (raft.SnapshotStore, error) {
+	conf := config.DefaultOptions
+	snStore, err := getSnapShotStore("memory", conf)
+	if err != nil {
+		return nil, err
+	}
+	return snStore, nil
 }
 
-func newSnapshot() raft.SnapshotStore {
-	return &snapshot{}
-}
-
-func (s snapshot) Create(version raft.SnapshotVersion, index, term uint64, configuration raft.Configuration, configurationIndex uint64, trans raft.Transport) (raft.SnapshotSink, error) {
-	//TODO implement me
-	panic("implement me")
-}
-
-func (s snapshot) List() ([]*raft.SnapshotMeta, error) {
-	//TODO implement me
-	panic("implement me")
-}
+func getSnapShotStore(snapshotStore string, conf config.Options) (raft.SnapshotStore, error) {
+	snapshotStoreDir := path.Join(conf.DirPath, "snapshot")
 
-func (s snapshot) Open(id string) (*raft.SnapshotMeta, io.ReadCloser, error) {
-	//TODO implement me
-	panic("implement me")
+	switch snapshotStore {
+	case "memory":
```

Comment:
As mentioned by [pr](https://github.com/ByteStorage/FlyDB/pull/197), corresponding fields can also be added to the store to achieve user-defined requirements.